app/src/components/common/outcomes/index.tsx

```diff
@@ -209,45 +209,31 @@ const Outcomes = (props: Props) => {
     )
   }
 
+  const uniform = (outcomes: Outcome[]): Outcome[] => {
+    return outcomes.map(o => {
+      o.probability = 100 / outcomes.length
+      return o
```

Comment:
I'd rather avoid mutation here:

```
return outcomes.map(o => ({
  ...o,
  probability = 100 / outcomes.length
}))
```

(or something like that; didn't test it).